superset-frontend/package.json

```diff
@@ -16,7 +16,7 @@
     "prod": "node --max_old_space_size=4096 ./node_modules/webpack/bin/webpack.js --mode=production --colors --progress",
     "build-dev": "cross-env NODE_OPTIONS=--max_old_space_size=8192 NODE_ENV=development webpack --mode=development --colors --progress",
     "build": "cross-env NODE_OPTIONS=--max_old_space_size=8192 NODE_ENV=production webpack --mode=production --colors --progress",
-    "lint": "eslint --ignore-path=.eslintignore --ext .js,.jsx,.ts,.tsx .",
+    "lint": "eslint --ignore-path=.eslintignore --ext .js,.jsx,.ts,.tsx . && prettier --check '{src,stylesheets}/**/*.{css,less,sass,scss}'",
```

Comment:
does eslint enforce the prettier checks already? Maybe we should move them out of eslint and into prettier if we're running the prettier checks anyway?

Reply:
Eslint Pretties(?) and checks the Prettification(?!) of of JS/JSX/TS/TSX files. Prettier itself is being used directly on LESS/CSS files a la carte. So now this verifies them in much the same way.